lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -135,7 +135,7 @@ getBlockHeader = label "getBlockHeader" $
         return $ BlockHeader
             { version
             , contentSize
-            , slot = (SlotId slotId slotEpoch)
+            , slot = (SlotId slotEpoch slotId)
```

Comment:
What about using the record named fields to precisely avoid this :) ?

Reply:
You have a point.